### DISCUSSION TITLE
Miner

### DISCUSSION THREAD

**Discussion by Arkad00** [Chia-Network/chia-blockchain#5081] [Show and tell]
Dear friends . teach me. I manu, dig, mine, for about a month nothing comes to my wallet, not a cent, Where is my mistake? SD 280 gb HDD 600 gb . I created the plots. All well running balls are coming. Where is my mistake? When to expect income ?

**Comment by moshenskyDV** [comment]
600GB... dude, you'll wait for reward for a years... Almost no chance

**Comment by Arkad00** [comment]
Re: [Chia-Network/chia-blockchain] Miner (#5081)



Здравствуйте, Moshenskyi.

Dear friend thank you for the answer I am disabling the extraction. Teach me what minimum equipment is necessary to see, check the production, help with the calculation and I will buy powerful equipment. Sincerely from Russia.

Вы писали 15 мая 2021 г., 15:41:19:





600GB... dude, you'll wait for reward for a years... Almost no chance
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.





--
С уважением,
 Fins                         ***@***.***

**Comment by Arkad00** [comment]
Re: [Chia-Network/chia-blockchain] Miner (#5081)



Здравствуйте, Moshenskyi.
Friend please help with the calculation of the Formula Condition (example) . Chia price 1000$ what equipment is needed to get 500$ per month . I understand that this will not be accurate, please help, until the equipment has become more expensive. Thanks, buddy.
Вы писали 15 мая 2021 г., 15:41:19:





600GB... dude, you'll wait for reward for a years... Almost no chance
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.





--
С уважением,
 Fins                         ***@***.***

**Comment by Isanderthul** [comment]
it is cheaper to buy chia than to buy hardware.
